### DISCUSSION TITLE
Onboarding PR isn't created

### DISCUSSION THREAD

**Discussion by BEGEMOT9I** [renovatebot/renovate#23736] [Request Help] [answered]
### How are you running Renovate?

Mend Renovate hosted app on github.com

### If you're self-hosting Renovate, select which platform you are using.

None

### Was this something which used to work for you, and then stopped?

It used to work, and then stopped

### Describe the problem

Hi! In our organization we are using renovate and it has been used in several repos before. But on Friday we created a new repo with renovate config (it was validated by the tool `renovate-config-validator`) and onboarding PR didn't appear in repo. There is the repo: https://github.com/adeo/ap--platformeco-cp-api-client. And we've already tried to recreate renovate config but it didn't help too. What we can do to fix this issue?

### Have you created a minimal reproduction repository?

I have explained in the description why a minimal reproduction is impossible

**Comment by HonkingGoose** [comment]
When your organization first onboarded Renovate, did the organization _select specific repositories_ where Renovate is allowed to run? Or did the organization give Renovate access to _all repositories_? Maybe Renovate is not allowed to run on your problem repository?

Also, does your repository have known package files: [^1]

> Renovate will ignore any repositories that don't have known package files, as well as any forks, so you can enable Renovate for all your repositories with no problems. That said, most people run Renovate on selected repositories. Unfortunately GitHub doesn't offer a "select all except X, Y, Z" option, so you must select each repository where you want Renovate to run.

And finally, is the problem-repository a _fork_? [^1]

> If you are using the Mend Renovate App then it has a custom behavior for forked repositories. If you choose "All repositories" when installing then forked repositories will be skipped by default, while if you choose "Select repositories" then they will be processed by Renovate even if they're a fork.

If none of the above helps you, read the [Renovate docs, troubleshooting guide](https://docs.renovatebot.com/troubleshooting/).

[^1]: [Renovate docs, onboarding hosted app](https://docs.renovatebot.com/getting-started/installing-onboarding/#hosted-githubcom-app)

**Comment by BEGEMOT9I** [comment]
> did the organization select specific repositories

Yes and now we are add them only when it's needed. So for this repo it was added

> does your repository have known package files

Yes, the renovate has been already used in the similar repo

> And finally, is the problem-repository a fork?

It is a repo

> If none of the above helps you, read the [Renovate docs, troubleshooting guide](https://docs.renovatebot.com/troubleshooting/).

We can't use a log manager bcs repo didn't appear in the list

**Reply by HonkingGoose** [reply]
When did you commit the Renovate config to the repository.? It can take some time for Renovate to see the change and start working.

**Reply by BEGEMOT9I** [reply]
3 days ago

**Reply by BEGEMOT9I** [reply]
And new config was readded 4 hours ago

**Reply by HonkingGoose** [reply]
Oh okay. In that case I'm out of ideas. I'll let the maintainers help you. 🙂

**Reply by viceice** [reply]
try disable and re-enable the repo again on the GitHub renovate app

**Reply by BEGEMOT9I** [reply, marked as answer]
Mmm, It started working this morning even though nothing has changed. Thank you for your participation! @HonkingGoose @viceice

**Reply by HonkingGoose** [reply]
@BEGEMOT9I happy to hear Renovate is working now! 🥳
